api/blocks/blocks_test.go

```diff
@@ -87,7 +97,8 @@ func testGetBlockByHeight(t *testing.T) {
 }
 
 func testGetFinalizedBlock(t *testing.T) {
-	res, statusCode := httpGet(t, ts.URL+"/blocks/finalized")
+	res, statusCode, err := tclient.RawHTTPClient().RawHTTPGet("/blocks/finalized")
```

Comment:
possibly we can add test for justified block

Reply:
Yeah make sense to have that test, but perhaps it would be easier to introduce that test on the justified PR